test/unittests/eof_validation_test.cpp

```diff
@@ -343,19 +342,23 @@ TEST_F(eof_validation, EOF1_undefined_opcodes)
         if (opcode == OP_RETF)
         {
             // RETF can be tested in 2nd code section.
-            cont =
-                "EF0001 010008 02000200040001 040000 00 00800000 00000000 E3000100 "_hex + OP_RETF;
+            cont = eof_bytecode(callf(1) + OP_STOP).code(OP_RETF, 0, 0, 0);
         }
         else
         {
-            cont += static_cast<uint8_t>(opcode);
+            cont = eof_bytecode(push(1) + 16 * OP_DUP1 + Opcode{static_cast<uint8_t>(opcode)}, 17);
```

Comment:
This needs some cleanup

- `cont` declaration with bytecde before `if` is not neeed anymore
- then here something like this should work I think
```cpp
auto code = push(1) + 16 * OP_DUP1 + Opcode{static_cast<uint8_t>(opcode)};
if (!is_terminating)
  code += OP_STOP;
cont = eof_bytecode(code,  max(17, 17 + op_stack_change));
```